PurpleNut/urls.py

```diff
@@ -33,7 +33,9 @@
     url(r'^login/',auth_views.LoginView.as_view(template_name='users/login.html') ,name='login'),
     url(r'^logout/',auth_views.LogoutView.as_view(template_name='users/logout.html') ,name='logout'),
     url(r'^profile/',user_views.profile,name='profile'),
-    url('^profile_edit/', user_views.edit_profile, name = 'edit_profile')
+    url(r'^profile_edit/', user_views.edit_profile, name = 'edit_profile'),
```

Comment:
Merge conflict: Use, r'^profile/(\d+)/ and r'profile/edit/' from the master branch

Reply:
made changes